src/main/java/org/wildfly/swarm/plugin/repository/XmlDependencyElement.java

```diff
@@ -54,12 +55,17 @@ public String getElementAsString() {
         return elementAsString;
     }
 
+    public boolean isNormalDependency() {
+        return scope == null || !scope.equals("import");
```

Comment:
`!"import".equals(scope)` should work too ;)

Reply:
Haha good point, thanks!

Reply:
Fixed.